addons/l10n_ch/tests/test_gen_qrr_reference.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
```

Comment:
Github considers this file as entirely new. It'd be better if it understood the file is actually a renaming of test_gen_isr_reference. This way, the history will not be impacted on github for the tests you don't modify. With the current version, it'll seem as if you introduced all these tests.

==> use git mv